Services/Mail/classes/class.ilPDMailBlockGUI.php

```diff
@@ -98,6 +98,7 @@ public function getHTML(): string
 
         $this->getMails();
         $this->setData($this->mails);
+        $this->loadData();
```

Comment:
Why is this change necessary? Why should block implementations be responsible to call this?